compose-cluster/proxy/http_proxy.js

```diff
@@ -0,0 +1,81 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+const SERVER_PORT = 8600;
+const CLIENT_PORT = 5000;
+const CLIENT_HOSTNAME = 'neuralcr';
+const AUTH_URL = 'http://lfsinitial:8080/system/sling/info.sessionInfo.json';
+
+const http = require('http');
+const fetch = require('node-fetch');
+
+function checkAuthentication(opts, method_allow, method_deny) {
+	//Check if the cookies are valid
+	if ('cookie' in opts) {
+		fetch(AUTH_URL, {
+				method: 'GET',
+				headers: {
+					Cookie: opts['cookie']
+				}
+			})
+			.then(res => res.json())
+			.then((json) => {
+				if ('userID' in json) {
+					if (json['userID'] == 'admin') {
```

Comment:
One weird thing is that usernames are kind of case insensitive, meaning that I can login with `Admin`, and I would correctly be identified as the `admin` user, but the session info JSON will still report me as `Admin`. Better make this check case insensitive:

```suggestion
					if (json['userID'].toUpperCase() === 'ADMIN') {
```